modules/security-context-constraints-psa-synchronization.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: CONCEPT
+[id="security-context-constraints-psa-synchronization_{context}"]
+= Security context constraint synchronization with pod security standards
+
+In addition to the global pod security admission control configuration, {product-title} translates pod security profiles that use the `warn` and `audit` labels for every namespace.
+
+By using a controller, clusters examine `ServiceAccount` object permissions to apply security context constraints to each namespace. Security context constraints (SCCs) are mapped to pod security profiles based on their field values. Pod security admission control `warn` and `audit` labels are applied to the most privileged pod security profile in the namespace to prevent warnings and audit logging as pods are created. 
+
+This controller cannot use automatic namespace labels for pods that are created without an intermediary, like the Deployment controller. This restriction prevents the SCC privileges of a user from being used to create a pod. If possible, do not run pods directly.
```

Comment:
We should probably rephrase this, the labels aren't applied or "used" for pods, they are applied to namespaces based on a consideration of the namespace-local SA privileges.

"Direct pod application" without an intermediary "pod controller" cannot be considered.

```
This restriction prevents the SCC privileges of a user from being used to create a pod.
```
This is rather inaccurate, they are still used, only if they appear more permissive than the pod security admission enforcement level, the pod will still not be admitted. On the other hand, if the permissions are less permissive, they will be very much applied.

Reply:
```suggestion
The controller applies labels to namespaces based on consideration of namespace-local service account privileges.

Direct pod application without an intermediary pod controller cannot be considered.  If possible, do not run pods directly.

The SCC privileges of a user are only used for pod creation if they are less permissive than the pod security admission enforcement level. If those privileges are less permissive, they are applied.

```

So, something like this? @stlaz

Reply:
Just leave out all of
```
Direct pod application without an intermediary pod controller cannot be considered. 

If possible, do not run pods directly. The SCC privileges of a user are only used for pod creation if they are less permissive than the pod security admission enforcement level. If those privileges are less permissive, they are applied.
```
Maybe in favour of something like:
```
Note that applying pod directly might leverage SCC privileges of the user running the pod, however user privileges are not considered during this automatic labelling.
```

Reply:
:+1: